changelog/7307.trivial.rst

```diff
@@ -0,0 +1,3 @@
+Progress from :meth:`sunpy.net.jsoc.JSOCClient.fetch` is now reported through the logger.
```

Comment:
Does this mean no progress bar any more? Does the progress show by default? How do I turn on and off the progress now compared to before? (are all questions that I would probably ask as a user looking at this change)

Reply:
All fair questions. Doing some manual testing, I find that the only way I'm able to control the verbosity of the progress bar is still through the `progress`keyword argument to `JSOCClient.fetch`. For example, the following code snippet still produces a progress bar, even though I've set the log level to "warn" (i.e. exclude "info"),

```python
import sunpy
sunpy.log.setLevel('WARN')

import astropy.units as u
from sunpy.net import jsoc
from sunpy.net import attrs as a
client = jsoc.JSOCClient()
response = client.search(a.Time('2014-01-01T00:00:00', '2014-01-01T00:00:10'),
                         a.jsoc.Series('aia.lev1_euv_12s'),
                         a.jsoc.PrimeKey('WAVELNTH','171'))
file = client.fetch(response, progress=True)
```
Given this, I'm confused as to what has actually changed on the `drms` side. Maybe I'm misunderstanding something here.

Reply:
If we cannot figure this out in the next few days, we can close this and just carry on with the `<0.7` pin that we already have. I don't really understand the point of `drms` v0.7 anyway...

Reply:
Drms with the progress keyword should print:

```
            if verbose:
                print(f"Downloading file {int(i + 1)} of {int(ndata)}...")
                print(f"    record: {di.record}")
                print(f"  filename: {di.filename}")
                print(f"  -> {os.path.relpath(fpath_new)}")

```
These are now logging.info